library-best-practices/JLBP-3.md

```diff
@@ -36,7 +38,7 @@ Definition of semantic versioning (SemVer): https://semver.org
     features) that is exposed through a library's public API
   - Adding a new class
   - Adding a new method
-- Special case: Merely bumping up the minimum required Java version (and not
-  making any breaking surface changes) does not necessarily mean that a
+- Special case: Merely increasing the upper-bound of the required Java version
```

Comment:
I believe the sentence was meant for "upper-bound" of the required version. Because bumping up minimum require Java version from 7 to 8, for example, would kill Java 7 users.

Reply:
This was correct as written. stet.